src/oms/core/util.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Get ISO8601 Date String
+ * @param d {Date}
+ * @returns string
+ */
+export const dateIsoString = (d: Date): string => {
+    const pad = (number: number) => {
+        if (number < 10) return `0${number}`;
+        return number;
+    };
+    return `${d.getUTCFullYear()}${pad(d.getUTCMonth() + 1)}${pad(d.getUTCDate())}T`
```

Comment:
Won't [toISOString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) be usefull in this case?